### ISSUE TITLE
NPM package has not been updated to 2.1.4

### ISSUE THREAD

**Issue by bensinclair** [homerours/cordova-music-controls-plugin#102]
When I run `cordova plugin add cordova-plugin-music-controls` it's installing version 2.0.0. 

The package has not been updated to 2.1.4: https://www.npmjs.com/package/cordova-plugin-music-controls

**Comment by fdambrosio** [comment]
I hope npm will be updated soon, I need to do: 
```
npm remove cordova-plugin-music-controls
npm install https://github.com/homerours/cordova-music-controls-plugin --save
ionic cordova plugin rm cordova-plugin-music-controls
ionic cordova plugin add https://github.com/homerours/cordova-music-controls-plugin
```

**Comment by BuddyLReno** [comment]
Yes, unfortunately, I don't have access to publish to NPM. Trying to rectify that. until then, 

`ionic cordova plugin add https://github.com/homerours/cordova-music-controls-plugin` will be what you need. :/

**Comment by fdambrosio** [comment]
there's news?

**Comment by Mokto** [comment]
FYI I created this : https://github.com/bhvr-business-solutions/cordova-plugin-audiocontroller, as this plugin is not updated anymore.

I rewrote the Objective C code to make it work with IOS 11.

**Comment by BuddyLReno** [comment]
@fdambrosio yep! just updated recently. 

@Mokto this plugin is still being actively developed and works with iOS 11. We had a snag with publishing permissions that have been unlocked. There's a few different people that can publish to NPM now.

**Comment by AshishMehra** [comment]
Well if @Mokto has re-written the Objective-C to upgrade the programming logic, then perhaps we should merge in the code and consolidate the two plugins into one better plugin that works everywhere :) ?

**Comment by BuddyLReno** [comment]
Agreed. We should review it and see what it will take.

**Comment by BuddyLReno** [comment]
@Mokto if you wouldn't mind, would you make a pull request from your fork and see if merging our code will produce a better plugin for everyone?

**Comment by Mokto** [comment]
@BuddyLReno I didn't know !

Give me a few days and I'll do it.

**Comment by BuddyLReno** [comment]
@Mokto awesome! The more the merrier!
